Sources/Defaults/Defaults.swift

```diff
@@ -65,6 +65,18 @@ public final class Defaults {
 			suite.removeObject(forKey: key)
 		}
 	}
+	
+	public func clear<T: Codable>(key: Defaults.Key<T>, suite: UserDefaults = .standard) {
+		for key in suite.dictionaryRepresentation().keys.filter({ $0 == key.name }) {
+			suite.removeObject(forKey: key)
+		}
```

Comment:
Why not just:

```swift
for key in keys {
	self[key] = key.defaultValue
}
```

?

Reply:
Good idea. Would this also work for `OptionalKey`?

Reply:
Yes: https://github.com/sindresorhus/Defaults/pull/21#issuecomment-519684656